pkg/services/datasources/accesscontrol.go

```diff
@@ -25,7 +25,6 @@ var (
 var (
 	// ConfigurationPageAccess is used to protect the "Configure > Data sources" tab access
 	ConfigurationPageAccess = accesscontrol.EvalAny(
-		accesscontrol.EvalPermission(accesscontrol.ActionDatasourcesExplore),
```

Comment:
Note: unrelated change, but I noticed that `Explore` is not located under data sources anymore, so we don't need to account for it here.